config/vmdb.tmpl.yml

```diff
@@ -169,6 +169,96 @@ policy_events:
     :purge_window_size: 1000
 product:
   maindb: ExtManagementSystem
+replication:
+  :exclude_tables:
+  - ar_internal_metadata
+  - assigned_server_roles
+  - audit_events
+  - binary_blobs
+  - binary_blob_parts
+  - chargeback_rate_details
+  - chargeback_rates
+  - conditions
+  - conditions_miq_policies
+  - configurations
+  - custom_buttons
+  - customization_specs
+  - database_backups
+  - event_logs
+  - file_depots
+  - jobs
+  - log_files
+  - metrics
+  - metrics_00
+  - metrics_01
+  - metrics_02
+  - metrics_03
+  - metrics_04
+  - metrics_05
+  - metrics_06
+  - metrics_07
+  - metrics_08
+  - metrics_09
+  - metrics_10
+  - metrics_11
+  - metrics_12
+  - metrics_13
+  - metrics_14
+  - metrics_15
+  - metrics_16
+  - metrics_17
+  - metrics_18
+  - metrics_19
+  - metrics_20
+  - metrics_21
+  - metrics_22
+  - metrics_23
+  - metric_rollups
+  - miq_actions
+  - miq_ae_classes
+  - miq_ae_fields
+  - miq_ae_instances
+  - miq_ae_methods
+  - miq_ae_namespaces
+  - miq_ae_values
+  - miq_ae_workspaces
+  - miq_alert_statuses
+  - miq_alerts
+  - miq_databases
+  - miq_enterprises
+  - miq_event_definitions
+  - miq_globals
+  - miq_groups
+  - miq_policies
+  - miq_policy_contents
+  - miq_product_features
+  - miq_queue
+  - miq_roles_features
+  - miq_report_result_details
+  - miq_report_results
+  - miq_reports
+  - miq_searches
+  - miq_servers_product_updates
+  - miq_sets
+  - miq_schedules
+  - miq_shortcuts
+  - miq_tasks
+  - miq_user_roles
+  - miq_widgets
+  - miq_widget_contents
+  - miq_workers
+  - rss_feeds
+  - scan_items
+  - schema_migrations
+  - server_roles
+  - sessions
+  - vim_performance_states
+  - vim_performance_tag_values
+  - vmdb_database_metrics
+  - vmdb_databases
+  - vmdb_indexes
+  - vmdb_metrics
+  - vmdb_tables
```

Comment:
I'd prefer to keep this in the original location in the config for now and not duplicate it.